src/CommunityToolkit/Xamarin.CommunityToolkit/Views/CameraView/iOS/FormsCameraView.ios.cs

```diff
@@ -239,6 +234,31 @@ public async Task TakePhoto()
 			}
 		}
 
+		AVCapturePhotoSettings GetCapturePhotoSettings()
+		{
+			var photoSettings = AVCapturePhotoSettings.Create();
+			photoSettings.FlashMode = (AVCaptureFlashMode)flashMode;
```

Comment:
This will be a merge conflict, I believe once https://github.com/xamarin/XamarinCommunityToolkit/pull/1522 is merged

Reply:
@AndreiMisiukevich thanks for checking, once #1522 is merged I can rebase and push again

Reply:
@AndreiMisiukevich it is good to go now, I have fixed the merge conflict 👍